nixos/modules/services/web-apps/docuseal.nix

```diff
@@ -0,0 +1,143 @@
+{ lib, pkgs, config, options, ... }:
+
+let
+  cfg = config.services.docuseal;
+
+  env = {
+    RAILS_ENV = "production";
+    NODE_ENV = "production";
+    WORKDIR = "/var/lib/docuseal";
+
+    PORT = toString (cfg.port);
+    HOST = cfg.host;
+  } // lib.optionalAttrs (cfg.database_url != null) {
+    DATABASE_URL = cfg.database_url;
+  } // cfg.extraConfig;
```

Comment:
What is the point of introducing the option `database_url` instead of using `extraConfig.DATABASE_URL` ?

Reply:
good idea. will do. I also try to use postgresql 15 for testing.

Reply:
fixed.